tests/TestCase/Heartbeat/Sensor/ConfigTest.php

```diff
@@ -60,6 +73,7 @@ public function testDefaultConfig()
         $this->assertEquals(true, $sensorConfig->getEnabled());
         $this->assertEquals(2, $sensorConfig->getSeverity());
         $this->assertEquals(null, $sensorConfig->getClass());
+        $this->assertEquals([], $sensorConfig->getSettings());
```

Comment:
This test is fine, but I think we should test the individual methods in their own test cases, too.
I'll do that once we merged this.